### PR TITLE
Added option to create a checksum file.

### PR DESCRIPTION
**Pull request by jamesiarmes**

### CODE REVIEW

File: lib/microbus/packager.rb [hunk 1]
```diff
@@ -50,11 +51,25 @@ def run # rubocop:disable MethodLength
...
+      checksum if opts.checksum
       file
     end
 
     private
 
+    # Generates a checksum of the build file and writes it to a new file.
+    #
+    # @return [String] MD5 checksum.
+    def checksum
```
Comment (pdrakeweb) [inline lib/microbus/packager.rb]: This method could probably use a more descriptive name.  It sounds as if it simply returns a value while the important bit is that it writes a file.

**Comment by pdrakeweb** [comment]
Looks good to me.

**Comment by pdrakeweb** [comment]
MD5 has the advantage of being available as the etag header on S3 HEAD requests I believe.

**Comment by jamesiarmes** [comment]
Yes, that's the reason I chose MD5 here.

**Pull request merged**